ui/console/src/main/scripts/plugins/accounts/plugins/accounts/ts/organizations.ts

```diff
@@ -19,8 +19,8 @@
 module HawkularAccounts {
 
     export var OrganizationsController = _module.controller("HawkularAccounts.OrganizationsController", [
-        '$rootScope', '$scope', 'HawkularAccounts.OrganizationService', '$log', '$location',
-        ($rootScope, $scope, OrganizationService, $log, $location) => {
+        '$rootScope', '$scope', '$log', '$location', 'HawkularAccount',
+        ($rootScope, $scope, $log, $location, HawkularAccount) => {
 
             $scope.organizations = [];
```

Comment:
Created https://issues.jboss.org/browse/HAWKULAR-240 for this, as this would require more changes than this isolated part.